client/memez/projects/memez/src/app/adapters/post.adapter.ts

```diff
@@ -1,24 +1,38 @@
-import {Injectable} from '@angular/core';
+import {Injectable}      from '@angular/core';
 import {BaseAjaxAdapter} from './base-ajax.adapter';
-import {IPost}           from '../types/Entities/IPost';
+import {IPost}           from '../../../../../../../shared/types/Entities/IPost';
 import {HttpClient}      from '@angular/common/http';
 
 
 @Injectable({
-    providedIn: 'root'
+  providedIn: 'root'
 })
 export class PostAdapter extends BaseAjaxAdapter {
-    
-    
-    constructor(
-        http: HttpClient
-    ) {
-        super(http);
-        window['PostAdapter'] = this;
-    }
-    
-    async getPosts(): Promise<IPost[]> {
-        return this.request<IPost[]>('posts');
-    }
-    
+
+
+  constructor(
+    http: HttpClient
+  ) {
+    super(http);
+    window['PostAdapter'] = this;
```

Comment:
you have some inconsistencies with casing. Sometimes you have camel case and other times you have title case